src/App.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div id="app">
     <header>
-      <toolbar />
+      <toolbar v-if="toolbarShown" />
```

Comment:
the toolbar can be shown in the login screen i think, because the toolbar has the actual toolbar and then tabs. you should probably see the "agriboard' toolbar, but not the tabs. i had implemented stubs for this in the toolbar component

Reply:
or maybe pass it into the component as a prop

Reply:
I think when you show a login page you aren't supposed to show the toolbar, isn't it usually just a full page login with a nicer color than the rest of the app?